code/game/turfs/simulated/floor/light_floor.dm

```diff
@@ -69,14 +69,13 @@
 	set_light(0)
 	..()
 
-/turf/simulated/floor/light/attack_hand(mob/user)
+/turf/simulated/floor/light/attack_hand()
```

Comment:
Keep the param here as it's defined on it's parent. Not using it is no issue but omitting it here can lead to issues down the line

Reply:
What about for line 77, the `params`?

Reply:
Same stuff.
![image](https://user-images.githubusercontent.com/15887760/93260718-0e020f00-f7a2-11ea-8e96-0811f8df4cb6.png)
The proc is defined with it so it's best to keep atleast the parameters given by the parents proc and above.

It can break if for example this happens:
```dm
/proc/test()
   attack_hand(user = usr)
```
If it calls the attack_hand on the light turf then it will runtime due to the argument missing